app/src/app/admin/articles/article/article.ts

```diff
@@ -70,7 +70,7 @@ namespace app.admin.articles.article {
                         return articleService.getArticle($stateParams.permalink)
                             .then((article) => {
 
-                                article._articleMetas = articleService.hydrateMetaFromTemplate(article._articleMetas, ArticleConfig.articleMetaTemplate);
+                                article._articleMetas = articleService.hydrateMetaFromTemplate(article, ArticleConfig.articleMetaTemplate);
```

Comment:
The function requires the article ID to hydrate the meta tags so I figured it would be better to pass the whole article rather than have an additional parameter to call the function with.